content-src/components/Header/Header.js

```diff
@@ -8,7 +8,8 @@ const Header = React.createClass({
   render() {
     const props = this.props;
     const currentRoute = props.currentRoute || {};
-    return (<header className="head">
+    return (<header className="head" hidden={props.currentRoute.path === "/"}>
```

Comment:
@bryanbell decided that there's too much going on with the search, the header, and the other elements; in the future we might add a corner menu or something to link to the full timeline view